README.md

```diff
@@ -129,49 +30,45 @@ Settings
 By default all fixes are enabled. You can disable them individually with settings:
 
 ```python
-with Configurator() as config:
-    config.add_settings({
-        # Don't check for badly declared forms.
-        "pyramid_sanity.check_form": False,
+config.add_settings({
+    # Don't check for badly declared forms.
+    "pyramid_sanity.check_form": False,
 
-        # Don't check for badly encoded query params.
-        "pyramid_sanity.check_params": False,
+    # Don't check for badly encoded query params.
+    "pyramid_sanity.check_params": False,
 
-        # Don't check for badly encoded URL paths.
-        "pyramid_sanity.check_path": False,
+    # Don't check for badly encoded URL paths.
+    "pyramid_sanity.check_path": False,
 
-        # Don't safely encode redirect locations.
-        "pyramid_sanity.ascii_safe_redirects": False
-    })
-    config.include("pyramid_sanity")
+    # Don't safely encode redirect locations.
+    "pyramid_sanity.ascii_safe_redirects": False
+})
 ```
 
 You can set `pyramid_sanity.disable_all` to `True` to disable all of the fixes,
 then enable only certain fixes one by one:
 
 ```python
-with Configurator() as config:
-    config.add_settings({
-        # Disable all fixes.
-        "pyramid_sanity.disable_all": True,
+config.add_settings({
+    # Disable all fixes.
+    "pyramid_sanity.disable_all": True,
 
-        # Enable only the badly encoded query params fix.
-        "pyramid_sanity.check_params": True,
-    })
-    config.include("pyramid_sanity")
+    # Enable only the badly encoded query params fix.
+    "pyramid_sanity.check_params": True,
+})
 ```
 
 Exceptions
 ----------
 
-`pyramid-sanity` triggers 400 Bad Request responses by returning
-`pyramid.httpexceptions.HTTPBadRequest` subclasses.
-
 All exceptions returned by `pyramid-sanity` are subclasses of
-`pyramid_sanity.exceptions.SanityException` (which is a subclass of
-`HTTPBadRequest`) and different exception subclasses are returned for different
-problems, so you can register
-[custom exception views](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/views.html#custom-exception-views)
+`pyramid_sanity.exceptions.SanityException`, which is a subclass of
+`pyramid.httpexceptions.HTTPBadRequest`.
```

Comment:
These two paragraphs were kind of saying the same thing about inheritance. So now one covers inheritance and the other points out the consequence of this (400's)